src/entity/SourceMember.ts

```diff
@@ -38,6 +42,10 @@ export class SourceMember {
   @Column({ type: 'text' })
   @Index('IDX_source_member_referralToken', { unique: true })
   referralToken: string;
+
+  @Column({ type: 'jsonb', default: {} })
+  @Index('IDX_source_member_flags_showPostsOnFeed', { synchronize: false })
```

Comment:
There's no need to index this field by itself. It's better to index it with user id as they go together